titanium-scanner.xcodeproj/project.pbxproj

```diff
@@ -214,6 +214,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 24DD6D1B1134B66800162E58 /* titanium.xcconfig */;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
```

Comment:
Is this really required?

Reply:
It was added automatically as I switched to yes and back to no